dds/DCPS/TypeObject.h

```diff
@@ -18,6 +18,10 @@
 #include <algorithm>
 #include <cstring>
 
+namespace DDS {
```

Comment:
Move this line below `OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL` to fix that Azure Pipelines error